docs/src/workflow examples/Basic Neural Architecture Search/Project.toml

```diff
@@ -0,0 +1,7 @@
+[deps]
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
+Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
```

Comment:
Not sure including `Literate` in the project is necessary or appropriate. See e.g. https://github.com/ablaom/MLJTutorial.jl/tree/dev/notebooks/01_data_representation .

Reply:
I did not know that one does not need to `add Literate` to use it. This is what I understood from what you said and after performing a quick experiment.

So far, two actions I need to be perform globally over the tutorials:
- Spell checking
- Ditching `Literate` from `Project.toml`